grafanalib/core.py

```diff
@@ -74,7 +75,7 @@ def to_json_data(self):
 FLOT = 'flot'
 
 ABSOLUTE_TYPE = 'absolute'
-DASHBOARD_TYPE = 'dashboard'
+DASHBOARD_TYPE = Literal['dashboards', 'link']
```

Comment:
Should this be `dashboard`, as this is what is set as the default value in the Class?

```suggestion
DASHBOARD_TYPE = Literal['dashboard', 'link']
```

Reply:
Ah, good catch; actually it's the default that should be changed, attached here is a screenshot of some actual dashboard json generated using the Grafana UI. That's my bad!

I'm happy to add some tests, will get on that asap. 
<img width="781" alt="Screenshot 2023-09-11 at 15 27 53" src="https://github.com/weaveworks/grafanalib/assets/3513885/61176910-bd96-4a1b-bf90-e5edccadb72a">